site/training/fake-tour/product-tour.qmd

```diff
@@ -16,7 +16,7 @@ format:
     chalkboard: false
     preview-links: auto
     view-distance: 2
-    logo: /validmind.png
+    logo: /favicon.svg
```

Comment:
@validbeck pushed one more commit, I think we want the favicon in the lower right, not the logo.